app/src/matchmaking/player.hpp

```diff
@@ -14,9 +14,7 @@ class Player {
     // The timeout threshold for the read engine command.
     // This has nothing to do with the time control itself.
     [[nodiscard]] std::chrono::milliseconds getTimeoutThreshold() const {
-        if (engine.getConfig().limit.nodes != 0     //
-            || engine.getConfig().limit.plies != 0  //
-            || time_control_.isFixedTime()) {
```

Comment:
mh why remove this?

Reply:
we have a time control for fixed time, that conditional is to check if there is a time control set